src/screens/token/components/delegate-token/components/validators/validator-card/validator-card.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { View, TouchableHighlight } from 'react-native';
+import stylesProvider from './styles';
+import { IThemeProps, withTheme } from '../../../../../../../core/theme/with-theme';
+import { SmartImage } from '../../../../../../../library/image/smart-image';
+import { BASE_DIMENSION, normalize } from '../../../../../../../styles/dimensions';
+import { Icon } from '../../../../../../../components/icon';
+import { smartConnect } from '../../../../../../../core/utils/smart-connect';
+import { Text } from '../../../../../../../library';
+import { IStatValue, IStatValueType } from '../../../../../../../core/blockchain/types/stats';
+import { NavigationService } from '../../../../../../../navigation/navigation-service';
+import { moonletValidator } from '../../../../../../../core/blockchain/celo/stats';
+import { Blockchain } from '../../../../../../../core/blockchain/types/blockchain';
+import { chainLayerValidator } from '../../../../../../../core/blockchain/cosmos/stats';
+
+export interface IExternalProps {
+    icon: string;
+    labelName: string;
+    rank: string;
+    website: string;
+    rightTitle: string;
+    rightSubtitle: string;
+    actionType: string;
+    bottomStats: IStatValue[];
+    blockchain: Blockchain;
+}
+
+export function getValueString(stat: IStatValue) {
+    switch (stat.type) {
+        case IStatValueType.STRING:
+            return stat.data.value;
+        case IStatValueType.AMOUNT:
+            return stat.data.value + ' ' + stat.data.tokenSymbol; // TODO format text based on blockchain
+    }
+}
+
+export const ValidatorCardComponent = (
+    props: IExternalProps & IThemeProps<ReturnType<typeof stylesProvider>>
+) => {
+    return (
+        <TouchableHighlight
+            onPress={() => {
+                NavigationService.navigate('Validator', {
+                    blockchain: props.blockchain,
+                    validator:
+                        props.blockchain === Blockchain.CELO
```

Comment:
is this necessary ? In all these components conditionals based on blockchain should be avoided

Reply:
It was just for dummy data not to confuse the Cosmos labels. Moved it in stats where there are multiples stats dummy